modules/apps/foundation/frontend-theme/space-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,216 @@
+$text: #74808A;
+
+$xxl-size: 6.2rem;
+$xl-size: 4.8rem;
+$l-size: 3.2rem;
+$m-size: 1.6rem;
+$s-size: 1.2rem;
+$desktop-breakpoint: 45rem;
+
+#wrapper {
+    background-color: $background;
+    color: $off-black;
+    margin: 0 auto;
+    padding:0;
+    position:relative;
+    
+        h1 {
+            color: $off-black;
+            display: inline-block;
+            font-family: Helvetica, Arial;
+            font-size: $xxl-size;
+            font-weight: 1000;
+            margin-bottom: 10px;
+            margin-top: 10px;
+            vertical-align: middle;
+            width: 300px;
+        }
+
+        #heading {
+            background-color: $white;
+        }
+
+        #navigation {
+            background-color: $background;
+            height: 100px;
+
+            .container-fluid {
+                padding-left: 37px;
+                padding-right: 37px;
+                background-color: $white;
+                position: fixed;
+                width: 100%;
+                z-index: 1000;
+            }
+    
+            ul {
+                background-color: $white;
+                display: inline-block;
+                margin: 0px;
+                overflow: hidden;
+                padding: 0px;
+                padding-top: 5px;
+                vertical-align: middle;
+                width: 100%;
+                width: 1193px;
+    
+                li {
+                    float: left;
+                    a {
+                        color: $off-black;
+                        display: block;
+                        margin: 0px;
+                        padding: 14px 16px;
+                        font-family: Open Sans, Helvetica, Arial;    
+                        font-size: 2.6rem;
+                        text-align: center;
+                        text-transform: uppercase;
+                        &:hover {
+                            color: $light-blue;
+                            text-decoration: none;
+                        }
+                    }
+                }  
+            }
+
+            @media screen and (max-width: 900px) {
+                h1 {
+                    display: block;
+                }
+
+                ul {
+                    list-style-type: none;
+                    margin: 0;
+                    padding: 0;
+                    width: 200px;
+
+                  li a {
+                      display: block;
+                      padding: 8px 16px;
+                      text-decoration: none;
+                  }
+                }
+            }
+        }
+
+    #content {
+        background-color: $background;
+        border: $dark-blue;
+        border-width: 3px;
+        color: $white;
+
+        #main-content {
+            border: $dark-blue;
+            border-width: 3px;
+
+            .portlet {
+                background-color: $light-blue;
+                border: $dark-blue;
+                border-width: 3px;
+            }
```

Comment:
You define your ```border``` and ```border-width``` as the same thing in all the 3 CSS blocks above. WE should sass them all together and just add what changes actually need to be there.